src/Spectre.AlgorithmsCli.Tests/Results/DivikResultTests.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "../../Spectre.libDivik/DivikResultStruct.h"	//TODO
```

Comment:
Nope. For some reason even the `$(SolutionDir)` from property sheet isn't getting caught.